deploy/openshift/ocp.sh

```diff
@@ -41,6 +41,12 @@ export OPENSHIFT_USERNAME=${OPENSHIFT_USERNAME:-${DEFAULT_OPENSHIFT_USERNAME}}
 DEFAULT_OPENSHIFT_PASSWORD="developer"
 export OPENSHIFT_PASSWORD=${OPENSHIFT_PASSWORD:-${DEFAULT_OPENSHIFT_PASSWORD}}
 
+DEFAULT_IMAGE_PULL_POLICY="Always"
```

Comment:
Looks like it duplicates the following lines https://github.com/eclipse/che/blob/59230a930dde746678b69acef9ca689c0262e1c2/deploy/openshift/ocp.sh#L74-L75